src/components/HealthClimateComponents/BlackCarbon.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
I believe `GroundLevel`, `BlackCarbon`, and `Methane` form three columns of the overall `HealthClimate` component. Please look at [Support](https://github.com/CodeForAfricaLabs/sensors.AFRICA/blob/79815ba972a6dd7cff268f73f22e4770c75921a4/src/components/Support.js#L15) component to see how we've set width in a responsive way for a three columns component and [IndoorOutdoor](https://github.com/CodeForAfricaLabs/sensors.AFRICA/blob/79815ba972a6dd7cff268f73f22e4770c75921a4/src/components/AirComponents/IndoorOutdoor.js#L18) component for a 2 column component.

You can adjust them if you feel the need but ensure **ALL** 2/3 column components on the site have matching width so they flow vertically and they're mobile responsive (I believe all existing ones are already fixed and we only need to worry about new ones, but just double check).